js/render/draw_symbol.js

```diff
@@ -125,7 +125,8 @@ function drawSymbol(painter, layer, posMatrix, tile, elementGroups, prefix, sdf)
         texsize = [painter.glyphAtlas.width / 4, painter.glyphAtlas.height / 4];
     } else {
         painter.spriteAtlas.bind(gl, alignedWithMap || painter.options.rotating ||
-            painter.options.zooming || fontScale !== 1 || sdf || painter.transform.pitch);
+            painter.options.zooming || fontScale !== 1 || sdf || painter.transform.pitch ||
+            browser.devicePixelRatio !== painter.spriteAtlas.pixelRatio);
```

Comment:
thoughts on this?


        var mapMoving = painter.options.rotating || painter.options.zooming;
        var iconScaled = fontScale != 1 || browser.devicePixelRatio !== painter.spriteAtlas.pixelRatio;
        var iconTransformed = alignedWithMap || painter.transform.pitch;
        painter.spriteAtlas.bind(gl, sdf || mapMoving || iconScaled || iconTransformed);